www/doc/2.0/api-endpoints.html

```diff
@@ -227,5 +227,64 @@ <h2 id="mathjs">/api/mathjs</h2>
     <h2 id="localerror">/api/local-error</h2>
     <!--TODO--> Forthcoming.
 
+
+    <h2 id="cost">/api/cost</h2>
+
+    <details>
+    <summary>Example inputs & outputs</summary>
+
+    <p>Request:</p>
+
+    <pre>{
+        formula: &lt;FPCore expression&gt;,
+        sample: [point ... ]
+      }</pre>
+
+    <p>Response:</p>
+
+    <pre>{
+        cost: [cost]
+      }</pre>
+
+    <p>Specific Example: sqrt(x+1)-sqrt(x)</p>
+
+    <p>Request:</p>
+
+    <pre>{
+        formula: &lt;(FPCore (x) (- (sqrt (+ x 1)) (sqrt x)))&gt;,
+        sample: [ [[1], -1.4142135623730951] ]
+      }</pre>
+
+    <p>Response:</p>
+
+    <pre>{
+        cost: [13120]
+      }</pre>
+
+    <p>Specific Example Simplified: (x+1)-(x)</p>
```

Comment:
I would change this to `Lower-Cost Example:`

Reply:
![image](https://github.com/herbie-fp/herbie/assets/112049313/d74f18a7-c77c-445f-853a-b23374394d60)
![image](https://github.com/herbie-fp/herbie/assets/112049313/4c6f1a45-4505-46ba-a085-74d8d16a4006)

Reply:
Done!